extensions/ql-vscode/src/remote-queries/view/RemoteQueries.tsx

```diff
@@ -313,9 +320,16 @@ const AnalysesResults = ({
   return (
     <>
       <VerticalSpace size={2} />
-      <AnalysesResultsTitle
-        totalAnalysesResults={totalAnalysesResults}
-        totalResults={totalResults} />
+      <Box display="flex">
+        <Box flexGrow={1}>
+          <AnalysesResultsTitle
+            totalAnalysesResults={totalAnalysesResults}
+            totalResults={totalResults} />
+        </Box>
+        <Box>
+          <ExportButton text="Export all" onClick={() => exportResults()}></ExportButton>
```

Comment:
Embarrasingly trivial: This is more succinct, though either way is fine.

```suggestion
          <ExportButton text="Export all" onClick={exportResults}></ExportButton>
```

Reply:
Not too trivial at all 🙈 I'm always happy to see neater ways of writing things!